content/services/design-systems.md

```diff
@@ -3,14 +3,23 @@ title: Integrated Design Systems
 sub: Want to reduce technical debt and scale your applications?
 index: Design Systems
 image:
-  src: pages/stacy-design-systems-training.jpg
-  alt: Stacy teaching developers at an event
+  src: pages/design-system-hero.jpg
+  alt: font sizes and previews for OddBird.net
+action:
+  text: Talk with us »
+  url: /contact/
 summary: |
-  **OddBird provides design-system training and implementation**
-  -- including strategy & design audits, automated style guides & pattern
-  libraries, component libraries, integrated toolkits, and robust
-  front-end architecture. If you're a project manager, scrum master, or
-  team leader, why not make your job easier?
+  **We partner with clients
+  to build design systems**
+  that improve team communication,
+  speed up development,
+  and lay the foundation
+  for a scalable code base.
+  We’re experts at building systems
+  — helping your team in integrated tools
+  & daily workflows
+  to maintain consistent designs
+  across digital products over the long-term.
```

Comment:
I have trouble following this sentence, especially "helping your team in integrated tools..."